commands/music/music-trivia.js

```diff
@@ -173,13 +180,14 @@ module.exports = class MusicTriviaCommand extends Command {
               message.guild.triviaData.triviaScore.set(
                 msg.author.username,
                 message.guild.triviaData.triviaScore.get(msg.author.username) +
-                  1
+                  2
```

Comment:
Why did you change the number from 1 to 2 here?

Reply:
Again, I moved the entire block to the top :) It's just that git does very nice diffs. But the number changed because the test for both things is now in this first if statement, so if no one had guessed anything and you instantly get both, you get the two points.